themes/reference/templates/variables.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Global variables for templates
+weight: 50
+---
+# Global variables for templates
+Prestashop offers a list of predefined global variables to help you use relevant information in the Front End Smarty templates.
+
+The variables were set in [`classes/FrontController.php`](https://github.com/PrestaShop/PrestaShop/blob/8.1.x/classes/controller/FrontController.php):
+
+```php
+$templateVars = [
+'cart' => $this->cart_presenter->present($cart),
+	'currency' => $this->getTemplateVarCurrency(),
+	'customer' => $this->getTemplateVarCustomer(),
+	'language' => $this->objectPresenter->present($this->context->language),
+	'page' => $this->getTemplateVarPage(),
+	'shop' => $this->getTemplateVarShop(),
+	'core_js_public_path' => $this->getCoreJsPublicPath(),
+	'urls' => $this->getTemplateVarUrls(),
+	'configuration' => $this->getTemplateVarConfiguration(),
+	'field_required' => $this->context->customer->validateFieldsRequiredDatabase(),
+	'breadcrumb' => $this->getBreadcrumb(),
+	'link' => $this->context->link,
+	'time' => time(),
+	'static_token' => Tools::getToken(false),
+	'token' => Tools::getToken(),
+	'debug' => _PS_MODE_DEV_,
+];
+```
+*You can dig into the code to get more information about what the variables do. Here is the list with a short description:*
+
+## List of variables
+
+### User Cart
+* `{$cart.products}`: List of products contained
```

Comment:
Now that I see at it I think it would be better to just mention that there's a `$cart` variable and if you want to know all variables you can do:

```
{$cart|dump}
```

or see the corresponding `Presenter`, `CartPresenter`, the same goes for Customer, etc., otherwise, every time something changes in the presenter, we have to update the doc

Reply:
Totally agree! I will do that change